### PR TITLE
Add helper functions to set up SBML model for PEtab

### PR DESCRIPTION
**Pull request by dweindl**

### CODE REVIEW

File: petab/sbml.py [hunk 1]
```diff
@@ -98,3 +98,199 @@ def constant_species_to_parameters(sbml_model):
...
+    """
+
+    if not check_units:
+        sbml_document.setConsistencyChecks(
+            libsbml.LIBSBML_CAT_UNITS_CONSISTENCY, False)
+
+    has_problems = sbml_document.checkConsistency()
+    if has_problems:
+        print_sbml_errors(sbml_document)
+        print('WARNING: Generated invalid SBML model. Check messages above.')
```
Comment (yannikschaelte) [inline petab/sbml.py]: logger.warn? By default gives the same output, and is easier to write e.g. to file + screen.

I usually do

```python
import logging

logger = logging.getLogger(__name__)
```

Comment (yannikschaelte) [reply petab/sbml.py]: same for all other print commands.

File: petab/sbml.py [hunk 2]
```diff
@@ -98,3 +98,199 @@ def constant_species_to_parameters(sbml_model):
...
+
+    return not has_problems
+
+
+def print_sbml_errors(sbml_document: libsbml.SBMLDocument,
```
Comment (yannikschaelte) [inline petab/sbml.py]: log_sbml_errors?

File: petab/sbml.py [hunk 3]
```diff
@@ -98,3 +98,199 @@ def constant_species_to_parameters(sbml_model):
...
+                      minimum_severity=libsbml.LIBSBML_SEV_WARNING):
+    """Print libsbml errors"""
```
Comment (yannikschaelte) [inline petab/sbml.py]: parameter descriptions?

File: petab/sbml.py [hunk 4]
```diff
@@ -98,3 +98,199 @@ def constant_species_to_parameters(sbml_model):
...
+    rule.setVariable(assignee_id)
+    rule.setFormula(formula)
+
+    return rule
+
+
+def add_model_output(sbml_model: libsbml.Model,
+                     observable_id: str,
+                     formula: str,
+                     observable_name: None):
```
Comment (yannikschaelte) [inline petab/sbml.py]:
```suggestion
                     observable_name: str = None):
```

File: petab/sbml.py [hunk 5]
```diff
@@ -98,3 +98,199 @@ def constant_species_to_parameters(sbml_model):
...
+        sbml_model: Model to add output to
+        formula: Formula string for model output
+        observable_id: ID without 'observable_' prefix
+        observable_name: Any observable name
+    """
+
+    if observable_name is None:
+        observable_name = observable_id
+
+    prefixed_id = f'observable_{observable_id}'
```
Comment (yannikschaelte) [inline petab/sbml.py]: check whether parameter / assignment rule exists in model already?

Comment (dweindl) [reply petab/sbml.py]: I will leave that to the user. If libsbml will not complain, I will not complain. When modifying the model, it is generally a good idea to check the result afterwards, see respective functions added here.

**Pull request merged**